back/app/finders/ideas_finder.rb

```diff
@@ -5,17 +5,18 @@ def initialize(params, scope: nil, includes: [], current_user: nil)
     scope ||= _base_scope
     super(
       params,
-      scope: only_idea_inputs_scope(scope),
+      # scope: only_idea_inputs_scope(scope),
```

Comment:
This will be fixed before merging, it's a temporary mechanism to unblock work on the front-end.